lib/socketclusterclient/log.rb

```diff
@@ -44,4 +44,27 @@ def disable_logging
   def enable_logging
     initialize_logger
   end
+
+
+  #
+  # Method to set logging level
+  #
+  #
+  #
+  #
+  def set_logging_level(level)
+    # need to set the logging level, create warning if logging level is invalid
+  end
+
+
+  #
+  # Method Create logging context
+  #
+  #
+  #
+  #
+  def set_logging_context
+    # create logging context, which can carry logging information throughout the thread...
```

Comment:
add implementation about setting context

Reply:
@sacOO7 This method is not required as @logger saves logging information throughout the thread.